data/model/oauth.py

```diff
@@ -352,8 +352,13 @@ def lookup_access_token_by_uuid(token_uuid):
 
 def lookup_access_token_for_user(user_obj, token_uuid):
     try:
-        return OAuthAccessToken.get(
-            OAuthAccessToken.authorized_user == user_obj, OAuthAccessToken.uuid == token_uuid
+        return (
```

Comment:
We should grab this query and verify that the query plan is using indexes. I assume it's called often.

Reply:
All ForeignKeyField's are indexed automatically in peewee, and the uuid is too

Reply:
Great! Closing accordingly.